packages/service-library/src/servicelib/rabbitmq/_utils.py

```diff
@@ -51,7 +53,13 @@ async def wait_till_rabbitmq_responsive(url: str) -> bool:
 
 
 def get_rabbitmq_client_unique_name(base_name: str) -> str:
-    return f"{base_name}_{socket.gethostname()}"
+    # NOTE: below prefix is guaranteed to change each time the preocess restarts
+    # Why is this desiarable?
+    # 1. the code base makes the above assumption, otherwise subcscribers and consumers do not work
+    # 2. enables restartability of webserver during [re]deploys
+    prefix_create_time = f"{psutil.Process(os.getpid()).create_time()}".strip(".")[-6:]
```

Comment:
Q: where is the file and the random characters you refer in the description? is the description outdated?